_posts/2023-01-05-exploring-process-automation-opportunities.md

```diff
@@ -0,0 +1,60 @@
+
+Process automation has not yet become a popular research subject, which is why there are still common misconceptions regarding this subject. Most notably, researchers often refer robotic process automation (RPA) as the one and only form of process automation and state that these two terms as the same. RPA is just one form of process automation since any form of development which reduces the necessity of manual labor can be considered as process automation. This includes system integrations, process planning and software development. RPA has been the most studied form of process automation for few years, which is why it has gained majority of the focus. 
+
+## RPA
+
+Robotic Process Automation, or RPA, has nothing to do with mechanical robots used in different factories, but instead is a purely software-based solution, designed to mimic different tasks performed by human employees. Developing an RPA solution often starts with determining all the steps a human worker performs in a certain process, including every input and click of a button. All the potential exceptions and errors should also be documented. After scripting all the steps, the software robot can mimic the described process.
+
+The reason RPA solutions can be more productive than human workers in certain processes it that RPA solutions are faster, and they do not make any humane errors, like typos or misinterpretations. From developers’ point of view, creating RPA solutions can be fast and low effort since the solution is implementable to use existing systems without the necessity to alter said systems in any way. The RPA solution can do all the same tasks as a human worker, so access to required systems can be obtained in a comparable manner, as with human employee.
+
+The necessity of structured data and specific types of processes is the main shortcoming of RPA solutions. If the process has multiple exceptions and requires an elevated level of deduction, creating a reliable RPA solution can be next to impossible. Also, as the solutions require the processes to remain as they are, system updates and changes to the process workflow are likely to cause the solution to malfunction.
+
+## Low-code in process automation
+
+Low-code is a programming technique derived from the concepts of Rapid Application Development, where the programmer has the freedom to spend more time designing the functionality and aesthetics of a application, rather than the syntax of the code itself. Low-Code Development Platforms are based on graphical user interfaces, which allow for the users to create applications and automated workflows without writing any code. Low-code contributes to the universal shortage of professional software developers by lowering the entry barrier to create solutions and applications. This research focused on Microsoft Power Platform as the selected low-code development platform.
```

Comment:
another picture would be nice to show what the tools are like

Reply:
Added image presenting the power platform tools